apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-description-links/src/index.tsx

```diff
@@ -56,13 +58,16 @@ const addBlockSupportLinks = (
 
 	processedBlocks[ name ] = true;
 
-	if ( blockLinks[ blockName ] ) {
-		settings.description = createLocalizedDescriptionWithLearnMore(
-			String( settings.title ),
-			settings.description,
-			blockLinks[ blockName ]
-		);
-	}
+	/**
+	 * Some elements are children, but have their own url for Learn More, and we want to show those.
+	 */
+	settings.description = createLocalizedDescriptionWithLearnMore(
+		String( settings.title ),
+		settings.description,
+		childrenBlockLinksWithDifferentUrl[ name ]
+			? childrenBlockLinksWithDifferentUrl[ name ]
+			: blockLinks[ blockName ]
+	);
```

Comment:
This will overwrite description with null when `childrenBlockLinksWithDifferentUrl` and `blockLinks` don't have this block.

Reply:
Damn, I changed it at last just to have less code. Sent a fix for this 👍